common/Field_Dimensions.hpp

```diff
@@ -10,56 +10,58 @@
 #include <Geometry2d/Point.hpp>
 #include <Geometry2d/Polygon.hpp>
 #include <Geometry2d/Rect.hpp>
+#include <Geometry2d/Line.hpp>
 
 /// This class contains constants defining the layout of the field.
 /// See the official SSL rules page for a detailed diagram:
 /// http://robocupssl.cpe.ku.ac.th/rules:main
 struct Field_Dimensions {
-    inline float Length() const { return _Length; }
-    inline float Width() const { return _Width; }
+    float Length() const { return _Length; }
```

Comment:
That was actually part of PR #656.  The compiler automatically inlines small methods, so for 99% of cases, you shouldn't have to specify inline.